### PR TITLE
fix Cmake/compile without KDU, enable exrconv, add docker

### PR DESCRIPTION
**Pull request by michaeldsmith**
- add docker without KDU
- fix Cmake and build so everything compiles without (or with) KDU

### CODE REVIEW

File: src/lib/OpenEXRCore/internal_htk.cpp [hunk 1]
```diff
@@ -221,7 +221,7 @@ internal_exr_apply_htk (exr_encode_pipeline_t* encode)
 
 #else
 
-#error
+//#error
```
Comment (palemieux) [inline src/lib/OpenEXRCore/internal_htk.cpp]: ?

**Review by palemieux — CHANGES_REQUESTED** [review]
What is the commented line?

File: src/lib/OpenEXRCore/internal_htk.cpp (hunk 1 above)
Comment (michaeldsmith) [reply src/lib/OpenEXRCore/internal_htk.cpp]: the #error was preventing it from compiling without KDU, not sure if you wanted to keep it has //#error or just remove line 224 completely, so I just commented it out

**Comment by michaeldsmith** [comment]
> What is the commented line?

the #error was at the hook mapping 
internal_exr_undo_htk() --> internal_exr_undo_ht()
internal_exr_apply_htk() --> internal_exr_apply_ht()

image

File: src/lib/OpenEXRCore/internal_htk.cpp (hunk 1 above)
Comment (palemieux) [reply src/lib/OpenEXRCore/internal_htk.cpp]: Please remove if no longer needed!

**Pull request merged**